backend/cmd/headlamp.go

```diff
@@ -1365,6 +1377,35 @@ func (c *HeadlampConfig) addContextsToStore(contexts []kubeconfig.Context, setup
 	return setupErrors
 }
 
+// removeContextFromFile removes the context from the kubeconfig file.
+func removeContextFromFile(w http.ResponseWriter, contextName string) error {
```

Comment:
removeContextFromFile could be removeContextFromDefaultKubeConfig? To be more descriptive. With just File it wasn't obvious that it was acting on the defaultKubeConfigPersistenceFile.